build-info-api/src/main/java/org/jfrog/build/api/builder/ArtifactBuilder.java

```diff
@@ -164,4 +165,12 @@ public ArtifactBuilder addProperty(Object key, Object value) {
         properties.put(key, value);
         return this;
     }
+
+    public ArtifactBuilder addProperty(Map props) {
```

Comment:
Why is this method needed?
Should it be named addProperties?
Let's declare the Map it gets with the appropriate key and values types <String, ...>
Let's add a description for it.

The above also goes for the same method in the DependencyBuilder class.